src/commonMain/kotlin/fr/acinq/lightning/channel/ChannelData.kt

```diff
@@ -376,6 +376,7 @@ data class ChannelKeys(
     val paymentBasepoint: PublicKey = paymentKey.publicKey()
     val delayedPaymentBasepoint: PublicKey = delayedPaymentKey.publicKey()
     val revocationBasepoint: PublicKey = revocationKey.publicKey()
+    val temporaryChannelId: ByteVector32 = (ByteVector(ByteArray(33) { 0 }) + revocationBasepoint.value).sha256()
```

Comment:
It could make sense indeed, I'll have a look at it. In lightning-kmp it's really also because the single-funded case will disappear very soon, so it's easier to go in that direction.

Reply:
Yeah, probably not very important.